src/deepsparse/server/config.py

```diff
@@ -119,6 +120,7 @@ def server_config_to_env(
     model_path: str,
     batch_size: int,
     integration: str,
+    num_cores: Optional[int] = None,
```

Comment:
as discussed, we don't want to add in `num_cores` at the server config level because this is also a property of the pipeline config. let's instead read in from the pipeline config where `num_cores` already exists.